### PR TITLE
mobile responsive

### PR DESCRIPTION
**Pull request by RicardoC23**
implement responsive mobile

### CODE REVIEW

File: src/components/AboutUs/BinlandInformation/InformationDetails/InformationDetails.style.ts [hunk 1]
```diff
@@ -1,46 +1,71 @@
+import { breakpoints } from 'shared/breakpoints'
 import styled from 'styled-components'
```
[the remark this answers is not in the document]
Comment (RicardoC23) [reply src/components/AboutUs/BinlandInformation/InformationDetails/InformationDetails.style.ts:2]: done

File: src/components/AboutUs/BinlandInformation/InformationDetails/InformationDetails.style.ts [hunk 2]
```diff
@@ -1,46 +1,71 @@
...
 import { colors } from 'styles/colors'
 
 export const ContainerDetails = styled.div`
   width: 100%;
-  height: 20.4rem;
```
[the remark this answers is not in the document]
Comment (RicardoC23) [reply src/components/AboutUs/BinlandInformation/InformationDetails/InformationDetails.style.ts:6]: done

File: src/components/AboutUs/BinlandInformation/BinlandInformation.styles.ts [hunk 3]
```diff
@@ -5,10 +5,14 @@ import Section from 'ui/Section/Section'
...
   display: flex;
+  align-items: flex-start;
   height: 100%;
   gap: 1.25rem;
+  width: 100%;
+  margin: 0;
 
   @media only screen and (min-width: ${breakpoints.lg}) {
     flex-direction: row;
+    margin: 9rem 0 0 0;
```
[the remark this answers is not in the document]
Comment (RicardoC23) [reply src/components/AboutUs/BinlandInformation/BinlandInformation.styles.ts:16]: done

File: src/components/AboutUs/BinlandInformation/InformationDetails/InformationDetails.style.ts [hunk 4]
```diff
@@ -1,46 +1,71 @@
...
   row-gap: 1.25rem;
   column-gap: 0.875rem;
   padding: 0 0.625rem;
+
+  @media only screen and (min-width: ${breakpoints.md}) {
+    grid-template-columns: repeat(2, 1fr);
+    height: 20.4rem;
+  }
+
+  @media only screen and (min-width: ${breakpoints.lg}) {
```
[the remark this answers is not in the document]
Comment (RicardoC23) [reply src/components/AboutUs/BinlandInformation/InformationDetails/InformationDetails.style.ts:18]: done

File: src/components/AboutUs/BinlandInformation/InformationDetails/InformationDetails.style.ts [hunk 5]
```diff
@@ -1,46 +1,71 @@
...
+    grid-template-columns: repeat(2, 1fr);
+    height: 20.4rem;
+  }
 `
 export const ItemDetails = styled.div`
-  height: 100%;
+  /* height: 100%; */
+  /* width: 150px; */
```
[the remark this answers is not in the document]
Comment (RicardoC23) [reply src/components/AboutUs/BinlandInformation/InformationDetails/InformationDetails.style.ts:25]: done

File: src/components/AboutUs/BinlandInformation/InformationDetails/InformationDetails.style.ts [hunk 6]
```diff
@@ -1,46 +1,71 @@
...
   flex-direction: column;
   justify-content: center;
+  align-items: flex-start;
+  color: ${colors.secondary.disabled};
+
+  @media only screen and (min-width: ${breakpoints.md}) {
+    height: 100%;
+    width: 100%;
+    /* width: 17.625rem;
+    height: 13.75rem; */
```
[the remark this answers is not in the document]
Comment (RicardoC23) [reply src/components/AboutUs/BinlandInformation/InformationDetails/InformationDetails.style.ts:39]: done

**Review by CrhistianT7 — APPROVED** [review]
Good Job!!!

**Pull request merged**